react/components/ProductAssemblyOptions/ProductAssemblyOptionItemQuantity.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react'
+import {
+  useProductAssemblyItem,
+  useProductAssemblyDispatch,
+} from '../ProductAssemblyContext'
+import { Checkbox, Radio, NumericStepper } from 'vtex.styleguide'
+import styles from './styles.css'
+
+const Single: FC = () => {
+  const { item, groupPath } = useProductAssemblyItem()
+  const dispatch = useProductAssemblyDispatch()
+  const selected = item.quantity === 1
+  return (
+    <div
+      onClick={() => {
+        if (item.quantity === 0) {
+          dispatch({
+            type: 'SET_QUANTITY',
+            args: {
+              itemId: item.id,
+              newQuantity: 1,
+              type: 'SINGLE',
+              groupPath,
+            },
+          })
+        }
+      }}
+    >
+      <Radio
+        checked={selected}
+        // Required but useless props
+        {...{ id: '', label: '', name: '', value: '', onChange: () => {} }}
```

Comment:
Do you really need to use the radio from styleguide?

Reply:
@klzns why not?